tests/filecheck/dialects/ltl/ltl_invalid.mlir

```diff
@@ -0,0 +1,13 @@
+// RUN XDSL_ROUNDTRIP
+
+// CHECK: attribute !ltl.sequence expected from variable 'T', but got !ltl.property
```

Comment:
Could you move this after the `module` or even better right after the statement that is invalid and raises this exception?

Reply:
Where is exception thrown from? I can't seem to find it in the dialect file.
Moreover, "variable 'T'" sounds like a Python-y generic message, which one shouldn't really care that much.

Could you please provide a human-legible error message that denotes which parts of the operation types should be in agreement?

Reply:
The message is given by the Annotated, should I use a custom verify_ instead?

Reply:
Hmm, let me have another look.

Reply:
I think it's given by the `ConstraintVar` via PyRDL.
Anyway, that is fine then.